tools/tests/tester/TestsRunner.test.ts

```diff
@@ -0,0 +1,27 @@
+import { read_yaml } from '../../helpers'
+import TestsRunner from '../../src/tester/TestsRunner'
+import { type OpenAPIV3 } from 'openapi-types'
+import { load_expected_evaluation, scrub_errors } from './helpers'
+
+test('stories folder', async () => {
+  // The password must match the one specified in .github/workflows/test-spec.yml
+  process.env.OPENSEARCH_PASSWORD = 'myStrongPassword123!'
```

Comment:
Why are we hard coding a password here, this should just be set in the workflow

Reply:
I should change it to `process.env.OPENSEARCH_PASSWORD =  process.env.OPENSEARCH_PASSWORD ?? 'myStrongPassword123!'`. The idea is streamlining local testing for the user. When they run `docker compose up`, this is the same password the cluster will be setup with. So, they can quickly run the tests right after the contain is set up and ready.

Reply:
It's not a security concern since this is just for testing and the password is already shown in the workflow.